src/pages/overzicht/Overzicht.jsx

```diff
@@ -0,0 +1,45 @@
+import './Overzicht.css';
+import posts from '../../constants/data.json';
+import formatDate from '../../helpers/formatDate.js';
+import {Link} from "react-router-dom";
+import Navigation from '../../components/navigation/Navigation';
+
+
+function Overzicht() {
+
+    return (
+        <>
+            <header><h1>Overzicht</h1></header>
+            <main>
+                <div className="outer-container">
+                    <h1>Overzicht Pagina</h1>
+                    <div className="inner-container">
+                        <article>
+                            {posts.map((item) => (
```

Comment:
je hebt hier item gebruikt, maar conventie is enkelvoud van de lijst dus zou dan post zijn.